packages/perseus/src/widgets/label-image.tsx

```diff
@@ -658,21 +668,31 @@ class LabelImage extends React.Component<LabelImageProps, LabelImageState> {
                     top: "bottom",
                     right: "left",
                     bottom: "top",
-                    center: "top",
-                }[LabelImage.imageSideForMarkerPosition(marker.x, marker.y)];
+                }[markerPosition];
             }
 
             const answerChoicesActive = index === activeMarkerIndex;
 
-            const answerStyles: CSSProperties = {
+            const adjustPopoverDistance: CSSProperties = {
                 [side]: 15, // move the popover closer to the marker
             };
 
+            const adjustPillDistance: CSSProperties = {
+                [`margin${
+                    markerPosition.charAt(0).toUpperCase() +
```

Comment:
I have to capitalize the first letter (i.e., `marginRight` to work as CSSProperties object.